src/articleView.js

```diff
@@ -0,0 +1,55 @@
+function getIDFromUrl() {
```

Comment:
There's not a lot of OO going on here - it's just a collection of functions. In such a small application that probably doesn't feel too bad, but in a larger application it would be pretty hard for another developer to come in and work out where all of these methods fitted into the larger application flow, and you'll also quickly start to feel pain when testing because it's difficult to isolated dependencies.